modules/administration/pages/reports.adoc

```diff
@@ -0,0 +1,89 @@
+[[reports]]
+= Generate Reports
+
+The [command]``spacewalk-report`` command is used to produce a variety of reports.
+These reports can be helpful for taking inventory of your subscribed systems, users, and organizations.
+Using reports is often simpler than gathering information manually from the {susemgr} {webui}, especially if you have many systems under management.
+
+To generate reports, you must have the [package]``spacewalk-reports`` package installed.
+
+The [command]``spacewalk-report`` command allows you to organize and display reports about content, systems, and user resources across {productname}.
+
+You can generate reports on:
+
+. System Inventory: list all the systems registered to {productname}.
+. Patches: list all the patches relevant to the registered systems and sorts patches by severity, as well as the systems that apply to a particular patch.
+. Users: list all registered users and any systems associated with a particular user.
+
+To get the report in CSV format, run this command at the command prompt on the server:
+
+----
+spacewalk-report <report_name>
+----
+
+This table lists the available reports:
+
+
+[[tab.bp.troubleshooting.spacewalk-report]]
+.[command]``spacewalk-report`` Reports
+[cols="1,1,1", options="header"]
+|===
+|Report | Invoked as | Description
+| Actions | [command]``actions`` | All actions.
+| Activation Keys | [command]``activation-keys`` | All activation keys, and the entitlements, channels, configuration channels, server groups, and packages associated with them.
+| Activation Keys: Channels | [command]``activation-keys-channels`` | All activation keys and the entities associated with each key.
+| Activation Keys: Configuration | [command]``activation-keys-config`` | All activation keys and the configuration channels associated with each key.
+| Activation Keys: Server Groups | [command]``activation-keys-groups`` | All activation keys and the server groups associated with each key.
```

Comment:
```suggestion
| Activation Keys: Server Groups | [command]``activation-keys-groups`` | All activation keys and the system groups associated with each key.
```

"System groups" is what we use in the WebUI and API. Documentation-wise "client groups" would be more appropriate, but I'd suggest sticking to the product jargon until we change it.